cognite/client/_api/raw.py

```diff
@@ -495,12 +495,9 @@ def retrieve(self, db_name: str, table_name: str, key: str) -> Row | None:
     def _make_columns_param(self, columns: list[str] | None) -> str | None:
         if columns is None:
             return None
-        if not isinstance(columns, List):
+        if not isinstance(columns, list):
             raise ValueError("Expected a list for argument columns")
-        if len(columns) == 0:
-            return ","
-        else:
-            return ",".join([str(x) for x in columns])
+        return "," if len(columns) == 0 else ",".join([str(x) for x in columns])
```

Comment:
I know not everyone is a fan of this so let me know if it should be reverted.

Reply:
OK, I will take a mix of your answer and @haakonvt s answer by reverting it, but also removing the list creation. Hope that is fine with both of ye.